pootle/apps/pootle_translationproject/utils.py

```diff
@@ -101,8 +103,27 @@ def clone_store(self, store, target_dir, update_cache=True):
         cloned.state = store.state
         cloned.filetype = store.filetype
         cloned.save()
+        self.clone_muted_checks(store, cloned)
         return cloned
 
+    def clone_muted_checks(self, source_store, target_store):
+        """Mute false positive checks in target store."""
+
+        false_positive_checks = QualityCheck.objects.filter(
+            unit__store=source_store,
+            false_positive=True
+        ).values('unit__unitid_hash', 'category', 'name')
+
+        for check in false_positive_checks:
+            QualityCheck.objects.filter(
+                unit__store=target_store,
+                unit__unitid_hash=check['unit__unitid_hash'],
```

Comment:
this would work

```
qs = QualityCheck.objects.none()
for check in false_positive_checks:
     qs = qs | QualityCheck.objects.filter...check_filters...)
qs.update(false_positive=True)
````